packages/database/test/path.test.ts

```diff
@@ -20,8 +20,8 @@ import { Path } from '../src/core/util/Path';
 
 describe('Path Tests', function() {
   const expectGreater = function(left, right) {
-    expect(Path.comparePaths(new Path(left), new Path(right))).to.equal(1);
-    expect(Path.comparePaths(new Path(right), new Path(left))).to.equal(-1);
+    expect(Path.comparePaths(new Path(left), new Path(right))).to.be.greaterThan(0)
```

Comment:
missing semicolon?

Reply:
Well, on GitHub it is missing. But I swear that I have this locally (via the Prettier commit that once again got stuck somewhere...)